content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md

```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
+
+### CNI
+
+There have been no changes to the deployment of `istio-cni`.
+
+### Pilot
+
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatibility, there are still some references to Pilot.
+
+## Mixer Deprecation
+
+`Mixer`, the process behind the `istio-telemetry` and `istio-policy` deployments, has been deprecated with the 1.5 release. `istio-policy` was disabled by default since Istio 1.3 and `istio-telemetry` is disabled by default in Istio 1.5.
+
+Telemetry is collected using an in-proxy extension mechanism (Telemetry V2) that does not require `Mixer`.
+
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by `mixer` have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
+
+If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.
+
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+### Feature Gaps between Telemetry V2 and Mixer Telemetry
+
+* Out of mesh telemetry is not supported. Some telemetry is missing if source or the destination of traffic is not sidecar injected.
+* Egress gateway telemetry is [not supported](https://github.com/istio/istio/issues/19385).
+* TCP telemetry is only supported with `mtls`.
+* Black Hole telemetry for TCP and HTTP protocols is not supported.
+* Histogram buckets are [significantly different](https://github.com/istio/istio/issues/20483) than Mixer Telemetry and cannot be changed.
+
+## Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or Istiod, will be used to connect to Istiod over port 15012.
+
+Note: despite the naming, in Istio 1.5 when `controlPlaneSecurityEnabled` is set to `false`, communication between the control plane will be secure by default.
```

Comment:
will this flag be removed in later versions. Looks like it doesn't matter what we set for this flag.

Reply:
Yes it will likely be removed, its only for backwards compatibility